### DISCUSSION TITLE
importing from stable in the settings when not detecting an previous osu! install, is broken

### DISCUSSION THREAD

**Discussion by g1-1-1** [ppy/osu#19502] [Q&A]
### Type

Game behaviour

### Bug description

fresh installing osu! lazer, ignoring import prompts in the setup wizard, and going to options to import all skins will bring me to a blank screen and automatically reports an unhandled error.

### Screenshots or videos

[github wouldn't let me submit this video directly, so i've uploaded it to streamable](https://streamable.com/ivwdjj)

### Version

2022.731.1-lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/9229157/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9229158/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9229159/updater.log)
[database.log](https://github.com/ppy/osu/files/9229160/database.log)

**Comment by g1-1-1** [comment]
oh wait, after looking at `runtime.log` it looks like it tries to open a drive i have bitlocker on, however the drive that osu! is on isn't on said drive.

**Comment by frenzibyte** [comment]
Sounds very relevant to https://github.com/ppy/osu-framework/issues/5306.

**Comment by frenzibyte** [comment]
Can you describe the way you have your folders structured in your system? Is the user profile folder mapped to the `Z:/` drive that osu! is trying to access?

**Reply by g1-1-1** [reply]
![explorer_kiJFOq2mNf](https://user-images.githubusercontent.com/24374615/182035566-40de7de7-a54f-4901-aa3a-8a87869efc0c.png)
osu! stable is in the games drive, everything relating to windows is in `C:\` still

**Reply by frenzibyte** [reply]
In other words, does inputting `%UserProfile%` in Windows Explorer lead you to the `Z:` drive?

**Reply by g1-1-1** [reply]
no, takes me to `C:\Users\kris`
